ReactiveCocoa/AppKit/NSButton.swift

```diff
@@ -0,0 +1,29 @@
+import ReactiveSwift
+import enum Result.NoError
+import AppKit
+
+extension Reactive where Base: NSButton {
+
+	internal var associatedAction: Atomic<(action: CocoaAction<Base>, disposable: Disposable?)?> {
+		return associatedValue { _ in Atomic(nil) }
+	}
+
+	public var pressed: CocoaAction<Base>? {
+		get {
+			return associatedAction.value?.action
+		}
+
+		nonmutating set {
+			associatedAction
+				.swap(newValue.map { action in
+					let disposable = CompositeDisposable()
+					disposable += isEnabled <~ action.isEnabled
+					disposable += trigger.observeValues { [unowned base = self.base] in
+						action.execute(base)
+					}
```

Comment:
This isn't compiling for me, giving me an error that `Binary operator '<~' cannot be applied to operands of type 'CocoaAction<Base>' and 'Signal<Base, NoError>'` but after looking at the `<~` function definition and the BindingTargetProtocol and BindingSourceProtocol, it seems like it should work to do that binding. That being said, doing `action <~ trigger.map {...` does not return a disposable right? So I'm not sure I understand how this should be added to the CompositeDisposable instance?